Source/BSGOutOfMemoryWatchdog.m

```diff
@@ -199,7 +200,7 @@ - (BOOL)computeDidOOMLastLaunchWithConfig:(BugsnagConfiguration *)config {
             BOOL sameVersions = [lastBootOSVersion isEqualToString:osVersion] &&
                                 [lastBootBundleVersion isEqualToString:bundleVersion] &&
                                 [lastBootAppVersion isEqualToString:appVersion];
-            BOOL shouldReport = (config.enabledErrorTypes & BSGErrorTypesOOMs)
+            BOOL shouldReport = (config.enabledErrorTypes.OOMs)
```

Comment:
Is it correct that we're only checking enabledErrorTypes and not autoDetectErrors?

Reply:
Nope, this is a redundant check which I'll remove: https://github.com/bugsnag/bugsnag-cocoa/blob/c3065a78666c13213f6a96f647caac83ea808fc0/Source/BugsnagClient.m#L534